autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -678,19 +681,20 @@ def _charge_usage(
         self,
         node_exec: NodeExecutionEntry,
         execution_count: int,
-    ) -> int:
+    ) -> Tuple[int, int]:
         total_cost = 0
+        remaining_balance = 0
         db_client = get_db_client()
         block = get_block(node_exec.block_id)
         if not block:
             logger.error(f"Block {node_exec.block_id} not found.")
-            return total_cost
+            return total_cost, remaining_balance
```

Comment:
nit: we can just set 0 here
So we don't misleadingly set `remaining_balance = 0` above

Also good for the static analysis to catch when remaining_balance is unintentionally unbounded